src/scancodes/set2/scancode.c

```
@@ -105,6 +135,233 @@ static inline uint8_t switch_e0_code(uint8_t code) {
+static void handle_state_init(uint8_t code) {
+    switch (code) {
+        case SC_ESCAPE_E0:
+            state = E0;
+            break;
+        case SC_ESCAPE_F0:
+            state = F0;
+            break;
+        case SC_ESCAPE_E1:
+            state = E1;
+            break;
+        case SC2_SPECIAL_F7:
```

Clang (14.0.6)

[warning] switch has 2 consecutive identical branches

(bugprone-branch-clone)